src/api.ts

```diff
@@ -126,6 +128,7 @@ export const apiFactory = (serviceId: number) => {
 				metrics.inc(Metrics.AuthFailuresByUuid, undefined, {
 					device_uuid: req.body.common_name,
 				});
+				await setTimeout(DELAY_ON_AUTH_FAIL)
```

Comment:
```suggestion
				await setTimeout(DELAY_ON_AUTH_FAIL);
```

Reply:
Yeah, the problem is that husky was updated to 8.x in this repo without updating to 8.x config - I've fixed that now